Userland/DevTools/HackStudio/HackStudioWidget.h

```diff
@@ -247,6 +248,7 @@ class HackStudioWidget : public GUI::Widget {
     RefPtr<GUI::Action> m_locations_history_back_action;
     RefPtr<GUI::Action> m_locations_history_forward_action;
     RefPtr<GUI::Action> m_toggle_semantic_highlighting_action;
+    RefPtr<GUI::Action> m_toggle_view_file_in_single_click;
```

Comment:
Please use the suffix `_action` here, similar to the other members.

Reply:
Updated function/variables name to match with Serenity OS style convention. :)